machine_management/creating_machinesets/creating-machineset-ibm-power-vs.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+[id="creating-machineset-ibm-power-vs"]
+= Creating a machine set on {ibmpowerProductName} Virtual Server
+include::_attributes/common-attributes.adoc[]
+:context: creating-machineset-ibm-power-vs
+
+toc::[]
+
+You can create a different machine set to serve a specific purpose in your {product-title} cluster on {ibmpowerProductName} Virtual Server. For example, you might create infrastructure machine sets and related machines so that you can move supporting workloads to the new machines.
```

Comment:
```suggestion
You can create a different compute machine set to serve a specific purpose in your {product-title} cluster on {ibmpowerProductName} Virtual Server. For example, you might create infrastructure machine sets and related machines so that you can move supporting workloads to the new machines.
```